### PR TITLE
Dala 1974_P2P_Upload_Page

### PR DESCRIPTION
**Pull request by emanueldfine**
# Pull Request
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one test exists testing the new feature
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] If there was a database entity class added, there must also be a migration script for creating the corresponding database if flyway is already used by the service
- [x] If there are changes done to the framework data models or to a database entity class, the following steps were completed
  - [x] The main branch is deployed to a dev server with `Reset non-user related Docker Volumes & Re-populate` turned on
  - [x] It's verified that the CD run is green
  - [x] Immediately after that the feature branch is deployed to the same server with `Reset non-user related Docker Volumes & Re-populate` turned off
  - [x] It's verified that the CD run is green
  - [x] The new feature is manually used/tested/observed on the dev server
- [x] The new version is deployed to the dev server "dev1" using this branch
  - [x] Run with setting `Reset non-user related Docker Volumes & Re-populate` turned on 
  - [x] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [x] It's verified that the CD run is green
  - [x] The new feature is manually used/tested/observed on dev server
  - [x] All implemented Social Logins have been tested manually in the UI
- [x] Make sure that pull requests in the development tools repository, which are connected to this pull requests, 
are merged

### CODE REVIEW

File: dataland-frontend/src/components/forms/CreateLksgDataset.vue [hunk 1]
```diff
@@ -227,7 +226,9 @@ export default defineComponent({
 
       const lksgDataset = (await lkSGDataControllerApi.getCompanyAssociatedLksgData(dataId)).data;
       const dataDateFromDataset = lksgDataset.data?.general?.masterData?.dataDate;
-      this.dataDate = dataDateFromDataset ? new Date(dataDateFromDataset) : this.dataDate;
```
Comment (MarcinStanczyk7) [inline dataland-frontend/src/components/forms/CreateLksgDataset.vue]: Why does it break the tests?

Comment (emanueldfine) [reply dataland-frontend/src/components/forms/CreateLksgDataset.vue]: It turned out that it does not break the tests. Some tests just accidentally failed after I pushed this change to remote. But actually this change was not the reason for it.
So we could change it to `this.dataDate = dataDateFromDataset ? new Date(dataDateFromDataset) : this.dataDate;` again if you like?

Comment (emanueldfine) [reply dataland-frontend/src/components/forms/CreateLksgDataset.vue]: (Currently this file `CreateLksgDataset.vue` is unchanged in the PR.)

Comment (MarcinStanczyk7) [reply dataland-frontend/src/components/forms/CreateLksgDataset.vue]: I suspect changes in Lksg in the near future, so maybe let's leave it as it is and change it during Lksg refactoring

**Pull request merged**